signalfx-org-metrics/metrics.yaml

```diff
@@ -159,100 +286,56 @@ sf.org.apm.numTroubleshootingMetricSets:
   description: |
     The cardinality of Troubleshooting MetricSets for each 1-minute time window.
 
-      * Dimension(s):  `orgId`
-      * Data resolution: 1 minute
-  metric_type: 
+        * Dimension(s): `orgId`
+        * Data resolution: 1 minute
+  metric_type: counter
   title: sf.org.apm.numTroubleshootingMetricSets
 
-sf.org.apm.numSpansDroppedInvalid:
-  brief: The number of invalid spans Splunk APM receives.
+sf.org.apm.grossContentBytesReceived:
+  brief: The volume of bytes Splunk APM receives off the wire before filtering
+    and throttling.
   description: |
-    The number of invalid spans Splunk APM receives. A span can be invalid
-    if there is no start time, trace ID, or service associated with it.
-    
-        * Dimension(s):  `orgId`
-        * Data resolution: 1 second
-  metric_type: counter
-  title: sf.org.apm.numSpansDroppedInvalid
+    The volume of bytes Splunk APM receives off the wire before filtering
+    and throttling. This content could be compressed.
 
-sf.org.apm.numSpansDroppedInvalidByToken:
-  brief: The number of invalid spans Splunk APM receives for a specific access token.
-  description: |
-    The number of invalid spans Splunk APM receives for a specific access token.
-    A span can be invalid if there is no start time, trace ID, or service associated with it.
-    
-        * Dimension(s):  `orgId`, `tokenId`
+        * Dimension(s): `orgId`
         * Data resolution: 1 second
   metric_type: counter
-  title: sf.org.apm.numSpansDroppedInvalidByToken
-
-sf.org.apm.numSpansDroppedOversize:
-  brief: The number of spans Splunk APM receives that are too large to process.
-  description: |
-    The number of spans Splunk APM receives that are too large to process.
-  
-      * Dimension(s):  `orgId`
-      * Data resolution: 1 second
-  metric_type: counter
-  title: sf.org.apm.numSpansDroppedOversize
-
-sf.org.apm.numSpansDroppedOversizeByToken:
-  brief: The number of spans Splunk APM receives that are too large to process
-    for a specific access token.
-  description: |
-    The number of spans Splunk APM receives that are too large to process
-    for a specific access token.
-  
-      * Dimension(s):  `orgId`, `tokenId`
-      * Data resolution: 1 second
-  metric_type: counter
-  title: sf.org.apm.numSpansDroppedOversizeByToken
+  title: sf.org.apm.grossContentBytesReceived
 
-sf.org.apm.grossContentBytesReceived:
-  brief: The number of bytes Splunk APM receives before filtering
+sf.org.apm.numContentBytesReceived:
+  brief: The volume of bytes Splunk APM accepts off the wire after filtering
     and throttling.
   description: |
-    The number of bytes Splunk APM receives before filtering
-    and throttling. The content might be compressed or uncompressed.
-  
-      * Dimension(s):  `orgId`
-      * Data resolution: 1 second
+    The volume of bytes Splunk APM accepts off the wire after filtering
```

Comment:
question: what happens if content is compressed? Is there another step a user should take if the content is compressed?

Reply:
Again, I don't know. Brian added this content.